app/views/passengers/_form.html.haml

```diff
@@ -49,20 +49,20 @@
       = form.check_box :active, id: :passenger_active
   = form.fields_for :doctors_note, @doctors_note do |d|
     .field
-      = d.label :expiration_date
+      = d.label 'expiration_date'
       = d.text_field :expiration_date,
-        class: 'datepicker', id: 'doctors_note_expiration_date'
+        class: 'datepicker', id: :doctors_note_expiration_date
     .field
-      = d.label :override_expiration_date
+      = d.label 'override_expiration_date'
       = d.check_box :override_expiration,
         id: 'check-click'
     - if @doctors_note.present?
-      .field#slide-date{class: passengers_form}
-        = d.label :enter_override_date
+      .field#slide-date{class: passengers_form_class}
+        = d.label 'enter_override_date'
         = d.text_field :override_until,
           class: 'datepicker'
   .field
-    = form.label :additional_notes
+    = form.label 'additional_notes'
```

Comment:
No, symbols are correct here. They should match the names of the columns of the model.

Reply:
In this case `:note`, and above `:override_expiration`

Reply:
Okay I see what you mean. I didn't realize you could change what would be displayed in the en.yml.